src/containers/TeamPage/DailyReportContainer.js

```diff
@@ -60,22 +60,23 @@ class DailyReportContainer extends Component {
 
```

Comment:
(Not related to This pull request directly )

Fixed this component based on the new daily report structure
- delete achievement
- Add tasks field